build/build.sh

```diff
@@ -35,10 +34,10 @@ pandoc --verbose \
   --csl=$CSL_PATH \
   --metadata link-citations=true \
   --mathjax \
-  --css=github-pandoc.css \
-  --include-in-header=build/assets/analytics.html \
-  --include-after-body=build/assets/anchors.html \
-  --include-after-body=build/assets/hypothesis.html \
+  --include-in-header=build/themes/default.html \
+  --include-in-header=build/plugins/analytics.html \
```

Comment:
> I think having all themes and plugins placed after-body would be good from a readability point of view, since most people viewing the raw HTML file are probably more interested in seeing the manuscript content rather than style/js additions.

I agree. Though I feel bad for anyone trying to read the raw HTML, as pandocs output is very messy and not formatted/indented very well. But I agree having all the HTML first and putting every "plugin" we can at the end is preferable.